src/Illuminate/Foundation/Application.php

```diff
@@ -151,7 +151,7 @@ class Application extends Container implements ApplicationContract, CachesConfig
      *
      * @var array
      */
-    protected $absoluteCachePathPrefixes = [DIRECTORY_SEPARATOR];
```

Comment:
Interesting, do you know why this? @owenvoke

Reply:
Looks like it's from this PR: https://github.com/laravel/framework/pull/32969

Config caching paths were breaking on Windows.